saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_subscription.py

```diff
@@ -1653,11 +1653,20 @@ def test_fulfillment_created(fulfillment, subscription_fulfillment_created_webho
     # given
     webhooks = [subscription_fulfillment_created_webhook]
     event_type = WebhookEventAsyncType.FULFILLMENT_CREATED
-    expected_payload = generate_fulfillment_payload(fulfillment)
+    expected_payload = generate_fulfillment_payload(
+        fulfillment, add_notify_customer_field=True
+    )
 
     # when
-    deliveries = create_deliveries_for_subscriptions(event_type, fulfillment, webhooks)
-
+    deliveries = create_deliveries_for_subscriptions(
+        event_type,
+        {
+            "order": fulfillment.order,
+            "fulfillment": fulfillment,
+            "notify_customer": True,
+        },
+        webhooks,
+    )
     # then
     assert json.loads(deliveries[0].payload.payload) == expected_payload
     assert len(deliveries) == len(webhooks)
```

Comment:
I would also suggest to add some tests to mutation which call the webhook, like `order_fulfill`. Here we test only the payload.